common/src/main/java/io/pravega/common/util/BufferView.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Your IDE has apparently reformatted the license header:

```
Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0
```

Reply:
OK. I'm pretty sure this is the cause why: #4083.

Reply:
@andreipaduroiu actually, we even have a checkstyle rule

```xml
<module name="RegexpHeader">
  <property name="header" value="/\*\*\n * Copyright \(c\)"/>
</module>
```

we can change the `value` to `/\*\n * Copyright \(c\)` and I volunteer to follow up the effort.

Reply:
I just noticed that one is not blocked to send a pull request before assigned to the issue. I will give it a try later today.